### ISSUE TITLE
Theme Detection Suggestion

### ISSUE THREAD

**Issue by Laronski** [OpenLiveWriter/OpenLiveWriter#306]
Hi, team,
I know a lot of [prior] effort was put into Windows Live Writer for theme detection [WordPress], but there are so many themes out there that are very complicated and they use plug-ins, cache managers and CSS, etc…

I'd like to put forth that it may be easier to have a process by which a person/user [of Open Live Writer] opens they blog [in a browser], opens a POST or PAGE, and uses a [separate app] routine to select [highlight] the area of the page that's just for the POST or PAGE.

The routine could grab the HTML, CSS, fonts, post width, etc. and create an Open Live Writer template from that as opposed to creating a temp post, publishing it and deleting it. 

I also think this would be more compatible with other blog services. By going this route, Open Live Writer functionality is increased, the core application isn't modified and the ancillary application can be updated without danager of messing something up. 

And at the very least, users should have the ability to change/set the default template width for composing posts/pages.

Thanks for your time.

**Comment by ScottIsAFool** [comment]
I know Brandon Turner did an app that sort of did this. Can you give this a try and see if it works for what you need? http://1drv.ms/1OmN9hV

**Comment by Laronski** [comment]
I had not seen that version, but it's not working for me. It's a little unclear on how to use it. 
I selected the title, the body and opened the HTML folder, but I see no way to save the results. The SAVE button never comes to be active.

As always, thanks.

**Comment by ScottIsAFool** [comment]
Yeah, it was always meant as a little utility not really meant for the broader public unless absolutely necessary :)

**Comment by Laronski** [comment]
I just tried it again. I think it's supposed to select the area, by highlighting it, but it's very close to what I was thinking. Sadly, it's throwing errors on me. This would be a great solution if it was refined a little. 

Can Brandon still be contacted; you believe he'd donate the code?

**Comment by ScottIsAFool** [comment]
I have the code, yes.

**Comment by Laronski** [comment]
I know you guys have plenty to do, but It's nice to have that code. Personally, I think should be refined a little and then live on separately. Thanks for the follow-up, Scott. I appreciate it…

**Comment by ScottIsAFool** [comment]
I could probably put the code up on Github, I'm not really sure what to do with it, @willduff may understand its code a bit better.

**Comment by willduff** [comment]
Definitely agree with the ideas here. Would love to see a feature like this. I could definitely take a look at Brandon's code and clean it up, though there are some more pressing bugs I want to get fixed first.

**Comment by ScottIsAFool** [comment]
@willduff let me know when you want to take a look and i'll dig out the code.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
